content/docs/capabilities/_cli-client-certificates.mdx

```diff
@@ -0,0 +1,29 @@
+import CodeBlock from '@theme/CodeBlock';
+
+### Client Certificates
+
+If Pomerium is configured to require client certificates, you will also need to provide a client certificate and private key when invoking the `pomerium-cli` command.
+
+You can specify these either by using PEM files, or (new in [v0.23.0](https://github.com/pomerium/cli/releases/tag/v0.23.0)) by searching for a certificate in the system trust store (on **macOS** and **Windows** only).
+
+To specify a client certificate and key using PEM files:
+
+<CodeBlock lang="bash">pomerium-cli {props.protocol} --client-cert cert.pem --client-key key.pem route.corp.example.com:1234</CodeBlock>
```

Comment:
This is a neat trick! I didn't know you could parameterize .mdx files like this.